data/projects.ini

```diff
@@ -164,6 +164,27 @@ git_url: https://github.com/boto/botocore.git
 description_short: Low-level, data-driven core of boto 3
 description_long: Low-level, data-driven core of boto 3
 
+[bulk_extractor]
```

Comment:
FYI: I've changed the name, since the github project name uses an underscore and added `dpkg_name` and `dpkg_source_name`